packages/remix-utils/src/usePageChange.test.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import {installGlobals} from '@remix-run/node';
+import {Link, Outlet} from '@remix-run/react';
+import {unstable_createRemixStub as createRemixStub} from '@remix-run/testing';
```

Comment:
Should we include your "createRemixStub" wrapper that you have in GST in this package as well?

Reply:
Yeah, I think so.

Went back and forth since it's such a light wrapper. But every app is going to need it.